packages/execution-environments/src/common/endowments/network.ts

```diff
@@ -92,6 +95,14 @@ const createNetwork = () => {
     return res;
   };
 
+  // TODO: The `ws` module for Node.js only partially implements `WebSocket`, so some functionality
```

Comment:
We may want to research which things are / aren't supported.

Reply:
Some details here: https://github.com/heineiuo/isomorphic-ws#limitations

I'm sure there are plenty of minute differences, but the major differences listed in the above link indicate that `ws` supersets the browser `WebSocket` API. We should consider preventing the Node-only features from being used.

Reply:
I wonder if any Node.js specific features are exposed since we wrap WebSocket anyway, it may not be.

Reply:
I think this should be fine for now.

Reply:
I think this should be fine for now.

Reply:
@Mrtenz Did you verify that all the APIs we use for the wrapper are present on `ws`?